diff -u libantlr3c-3.4/configure.ac libantlr3c-3.4-patched/configure.ac
--- libantlr3c-3.4/configure.ac	2011-07-19 19:14:04.000000000 +0100
+++ libantlr3c-3.4-patched/configure.ac	2015-08-03 15:45:09.630456136 +0100
@@ -51,7 +51,7 @@
 		[AS_HELP_STRING([--disable-antlrdebug], [Turns off default flags that include the antlr debugger in the runtime. Specify to remove debugger and the socket dependancies (default --enable-antlrdebug)])],
 		[], [enable_antlrdebug=yes])
 
-AM_INIT_AUTOMAKE(foreign)
+AM_INIT_AUTOMAKE([foreign subdir-objects])
 AC_LANG(C)
 AC_PROG_CC([xlc aCC gcc cc])
 AM_MAINTAINER_MODE
@@ -134,7 +134,7 @@
  ;;
 esac
 
-CFLAGS="$CCFLAGS64 $CPPCMNTFLAGS $OPTIMFLAGS $DEBUGFLAGS $WARNFLAGS"
+CFLAGS="$CFLAGS $CCFLAGS64 $CPPCMNTFLAGS $OPTIMFLAGS $DEBUGFLAGS $WARNFLAGS"
 AC_MSG_RESULT([$CFLAGS])
 AC_SUBST([OBJECT_MODE])
 
@@ -148,7 +148,6 @@
 # Checks for libraries.
 
 # Checks for header files.
-AC_INCLUDES_DEFAULT()
 AC_HEADER_RESOLV
 AC_CHECK_HEADERS([sys/malloc.h malloc.h], [], [],
 [[#ifdef HAVE_SYS_MALLOC_H
Common subdirectories: libantlr3c-3.4/doxygen and libantlr3c-3.4-patched/doxygen
Common subdirectories: libantlr3c-3.4/include and libantlr3c-3.4-patched/include
Common subdirectories: libantlr3c-3.4/m4 and libantlr3c-3.4-patched/m4
Common subdirectories: libantlr3c-3.4/src and libantlr3c-3.4-patched/src
Common subdirectories: libantlr3c-3.4/vsrulefiles and libantlr3c-3.4-patched/vsrulefiles
